src/server.ts

```diff
@@ -46,7 +46,7 @@ app.get('/results', (req, res) => {
 	try {
```

Comment:
This path/function doesn't work and isn't used, so should we delete it?

Reply:
idk why that even exists. ask Blake

Reply:
@arthurs ?

Reply:
i believe this was used as part of the spark integration originally. with the html file now being in gcloud it can be removed